alcohol_drugs_survey/config.json

```diff
@@ -25,7 +25,7 @@
                          "Russell Poldrack"
                         ], 
         "time":999,
-        "experiment_variables": "Cannabis questionnaire: The Cannabis Use Disorders Identification Test, Drug use questionnaire: DAST-10",
+        "experiment_variables": "Cannabis questionnaire: The Cannabis Use Disorders Identification Test - Revised, Drug use questionnaire: DAST-10",
```

Comment:
Meant to be under "notes". Other surveys are called an official-y name. This survey is composed of other official surveys so I included them in the notes.